internal/health/condition/check_ready.go

```diff
@@ -6,15 +6,52 @@ package condition
 
 import (
 	"context"
+	"strings"
```

Comment:
was there a rational to link split-brain with `ready` check? You can have a split brain but still be ready in a 3 member-cluster, right? Was a separate condition considered?

Reply:
> You can have a split brain but still be ready in a 3 member-cluster, right?

yes, but that's why druid controller is detecting this in etcd ready status, otherwise split-brain will go undetected as all are in ready state.

Reply:
yes, I also checked the etcd status:
```yaml
 - lastTransitionTime: "2025-09-29T06:00:40Z"
    lastUpdateTime: "2025-09-29T11:33:56Z"
    message: The majority of ETCD members is ready
    reason: Quorate
    status: "True"
    type: Ready
```
It says "majority of ETCD members is ready" which is also true in split-brain, now I got your point ... should we create a separate of may be `type:  ClusterReady` ?